### ISSUE TITLE
custom cell classes

### ISSUE THREAD

**Issue by Vasiliy-Bondarenko** [ratiw/vuetable-2#504]
Is there a way to set custom classes for each cell? i want to send classes from backend. something like:
```
{
   value,
   value_class,
   value2,
   value2_class,
}
```

**Comment by ratiw** [comment]
@Vasiliy-Bondarenko Not really. Unless you use special field. Are you using v1.7.x or v2.0-beta?

**Comment by Vasiliy-Bondarenko** [comment]
@ratiw special field? what do you mean? some special component?
I don't use any version just yet. i'm choosing which package is suitable for my needs :) i'm ok with beta if it's worth it.

**Comment by ratiw** [comment]
@Vasiliy-Bondarenko Then, go through the documentation [here](https://vuetable.com) first and see if you like it. Or ask some questions on what you're unclear after going through the documentation.
